src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="styles.css">
+  <!-- font awesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  
+  <title>Leaderboard</title>
+</head>
+
+<body>
+  <h1>Leaderboard</h1>
+  <div class="container">
+    <div class="left-section">
+      <div id="recentScoresContainer">
+        <h2 id="recentScoresHeading">Recent Scores</h2>
+        <button id="refreshButton"><i class="fa fa-refresh" aria-hidden="true"></i></button>
+      </div>
+      <div id="leaderboard"></div>
+    </div>
+    <div class="right-section">
+      <h2>Add Your Scores</h2>
+      <form id="scoreForm">
+        <input type="text" id="playerName" placeholder="Your name" required><br>
+        <input type="number" id="playerScore" placeholder="Your score" required><br>
+        <button type="submit">Submit</button>
+      </form>
+    </div>
+  </div>
+  
+  <div id="message"></div>
+
+  <script src="main.js" type="module"></script>
+</body>
```

Comment:
Your design
![image](https://github.com/Mike111222/MyLeaderBoard/assets/78251927/54721774-1289-4843-bf0e-c07a56f79f89)

Wireframe
![image](https://github.com/Mike111222/MyLeaderBoard/assets/78251927/4c666da6-fff7-45ff-b397-b54b95ae97e2)

Kindly ensure that your HTML code looks like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), according to the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md).

Hint.
-  You can use a placeholder text instead. (just to make the layout) 🤓

Reply:
I have done the changes you mentioned and I am really thankful for the kind words.